src/QsCompiler/Tests.Compiler/TestCases/QirTests/TestBuiltInIntrinsics1.ll

```diff
@@ -1,4 +1,4 @@
-define void @Microsoft__Quantum__Testing__QIR__TestBuiltInIntrinsics__body() #0 {
+define void @Microsoft__Quantum__Testing__QIR__TestBuiltInIntrinsics__body() {
```

Comment:
FYI
In some earlier PR I also have seen the [`internal` keyword](https://llvm.org/docs/LangRef.html#linkage-types) used.
Looks like it will not harm adding that too if appropriate.

Reply:
Exactly what I'm building towards :). You'll see that in the next PR, https://github.com/microsoft/qsharp-compiler/pull/1011.